src/app/handler/user/user.handler.go

```diff
@@ -34,27 +50,102 @@ func (h *Handler) FindOne(c *router.FiberCtx) {
 		return
 	}
 
-	c.JSON(http.StatusCreated, user)
+	c.JSON(http.StatusOK, &dto.ResponseSuccess{
+		StatusCode: http.StatusOK,
+		Message:    userconst.FindOneUserSuccessMessage,
+		Data:       user,
+	})
 	return
```

Comment:
Sorry, may be I missed something a bit. So after this, we will return success response as this struct. When frontend need to access the actual data, they need to call something like `response.data`?

Reply:
Sorry, I forgot to look at auth svc. Yeah, I think let's remove it and return as pure dto, much ligther. Let me check real quick